### ISSUE TITLE
TV guide channel header accessibility in grid mode

### ISSUE THREAD

**Issue by defagos** [SRGSSR/playsrg-apple#208]
Channel headers are currently not accessible via VoiceOver in grid mode. This could likely be fixed with collection-view specific mechanism for header accessibility management.

### Issue type

Incorrect behavior

### Environment information

* Application version: iOS 3.6.0 / tvOS 3.6.0
* iOS / tvOS version: _version_
* Device: _model_

### Reproducibility

Always reproducible

### Steps to reproduce

1. Open the Play SRF app and its TV guide.
1. Display the grid layout.
1. Enable VoiceOver.
1. Navigate.
